generator/sequence_transform_data.py

```diff
@@ -627,11 +622,11 @@ def generate_sequence_transform_data(data_header_file, test_header_file):
 
     for sequence, transform in seq_tranform_list:
         # Don't add rules with transformation functions to test header for now
-        if transform[-1] not in output_func_symbol_map:
-            transform_without_leading_wordbreak = transform[1:] if sequence.startswith(WORDBREAK_SYMBOL) else transform
-            test_rule = create_test_rule_c_string(symbol_map, sequence, transform_without_leading_wordbreak)
-            test_rule_c_sequences.append(test_rule[0])
-            test_rule_c_transforms.append(test_rule[1])
+        if len(transform) == 0 or transform[-1] not in output_func_symbol_map:
```

Comment:
I know you don't like rule_search testing, but this should be `!= 0` :D

Reply:
Oh, hah! That was obviously unintentional. Good catch :)